### DISCUSSION TITLE
Modal height

### DISCUSSION THREAD

**Discussion by dqii** [react-native-modal/react-native-modal#628] [Q&A]
When you set height in style props, the height doesn't apply. It still shows the full height modal. How do you change the height of the modal?

**Comment by anmol242** [comment]
@dqii Here is how you can do it:

JSX
```
return (
    <Modal isVisible={isModalVisible} style={styles.modalContainer}>
      <View style={styles.modalContent}>
        {/* do Something */}
      </View>
    </Modal>
  );
```
CSS:

```
 modalContainer: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
modalContent: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    display: 'flex',
    fontSize: 16,
    fontWeight: 500,
    backgroundColor: '#ffffff',
    maxHeight: 500,
    width: 400,
    borderRadius: 10,
  },
```
Adjust the maxHeight and width according to your needs

**Reply by timoisalive** [reply]
Seems like if your content is taller than your modal height on Android, the modal will not respect for example `maxHeight: '90%'` style.

We worked our way around by using `maxHeight: windowDimensions.height * 0.9`.
